src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.html

```diff
@@ -87,6 +87,15 @@
 <div *ngIf="exercise.type === ExerciseType.MODELING">
     <!-- ACTION START -->
     <div class="btn-group mr-2">
+        <button
+            jhi-exercise-action-button
+            buttonIcon="users"
+            [buttonLabel]="'artemisApp.exerciseActions.viewTeam' | translate"
+            *ngIf="exercise.teamMode && participationStatusWrapper() !== ParticipationStatus.NO_TEAM_ASSIGNED"
+            [smallButton]="smallButtons"
+            [hideLabelMobile]="true"
+            [routerLink]="['/course-management', courseId, 'exercises', exercise.id, 'teams', assignedTeamId]"
```

Comment:
Normal users should not be able to navigate to `course-management` URLs.
Can we offer the same page also on a different based on `courses/:courseId/exercises/...`

Reply:
Good point. I added a student-specific route at `courses/:courseId/exercises/:exerciseId/teams/:teamId` and restricted the other one to TAs and above.